Resources/Private/Templates/FrontEnd/FrontEnd.html

```diff
@@ -240,15 +240,8 @@
 <!-- ###SINGLE_VIEW###  -->
 
 <!-- ###FIELD_WRAPPER_TITLE### -->
-    <h2
-        <!-- ###FIELD_WRAPPER_IMAGE### -->
-            ###STYLE_SINGLEVIEWTITLE###
-        <!-- ###FIELD_WRAPPER_IMAGE### -->
-    >
-    <!-- ###FIELD_WRAPPER_EVENT_TYPE### -->
-        ###EVENT_TYPE###:
-    <!-- ###FIELD_WRAPPER_EVENT_TYPE### -->
-    ###TITLE###</h2>
+<h2><!-- ###FIELD_WRAPPER_EVENT_TYPE### -->###EVENT_TYPE###:<!-- ###FIELD_WRAPPER_EVENT_TYPE### -->###TITLE###</h2>
```

Comment:
Please also keep the formatting as before. This makes the subparts easier to see, and it will also preserve the whitespace between the `: ` (after the event type) and the title.